OSVRUnreal/Plugins/OSVR/Source/OSVR/Private/OSVREntryPoint.cpp

```diff
@@ -29,13 +29,33 @@ DEFINE_LOG_CATEGORY(OSVREntryPointLog);
 
 OSVREntryPoint::OSVREntryPoint()
 {
+}
+
+void OSVREntryPoint::Initialize()
+{
+    // avoid double initialization
+    if (osvrClientContext != nullptr)
+    {
+        // too spammy to log here
+        return;
+    }
+
+    // hack: to avoid delay on OSVR server connection attempt when not using OSVR, don't initialize if nohmd command line param specified
+    if (FParse::Param(FCommandLine::Get(), TEXT("nohmd")))

```

Comment:
If the developer is running OSVRInput, they are still an OSVR user even if the game is running in non-vr mode. OSVRInput implements the controller and motion controller interfaces in unreal, which are orthogonal to "vr mode". If the developer doesn't want either OSVRHMD or OSVRInput, the current workaround is to just disable the OSVR plugin, although in the future we may split OSVRInput into its own plugin, with a common OSVRClientKit module between them (i.e. move OSVREntryPoint to OSVRClientKit).